src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -286,8 +286,11 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
 
     val proxyGroupDeletion = userF.flatMap(gcsDAO.deleteProxyGroup)
 
+    val tokenRemoval = toFutureTry(gcsDAO.revokeToken(userRef) flatMap { _ => gcsDAO.deleteToken(userRef) })
```

Comment:
ok - but why for this operation and not others?  Maybe this is a face conversation.